### ISSUE TITLE
Black Screen at Chapter 11 of Danball Senki W v2.02 (PPSSPP v1.12.3 - PC)

### ISSUE THREAD

**Issue by ItsukaHiro** [hrydgard/ppsspp#15453]
I'll go straight: In a dungeon at Tiny Orbit (Chapter 11 - DLC), after clear it, a conversation will be shown. After that, a black screen. I can't do anything now.

I tried to redownload the game, delete and reinstall data,... but none of them work. Also did  it on a mobile device, and same result.

Another test at v1.11.2, and it's the same.

I noticed that the BGM still play, and when I press [Select] there is a sound mean the game still running.

Video: https://youtu.be/n_n7haD-VNA

**Comment by ANR2ME** [comment]
Based on https://discord.com/channels/293316141479362560/480103153803460614/949986949106647132
Someone said PPSSPP have issue with DLC on Gladiator Begins after https://github.com/hrydgard/ppsspp/commit/87dcf60ed037d5f2168d5f0033dda002b098e177

So you may want to try PPSSPP version before `v1.11.3-942-g87dcf60ed` in case this game also had similar issue too.

**Comment by ItsukaHiro** [comment]
> Based on https://discord.com/channels/293316141479362560/480103153803460614/949986949106647132
> Someone said PPSSPP have issue with DLC on Gladiator Begins after https://github.com/hrydgard/ppsspp/commit/87dcf60ed037d5f2168d5f0033dda002b098e177
> 
> So you may want to try PPSSPP version before `v1.11.3-942-g87dcf60ed` in case this game also had similar issue too.
> You can download old version of PPSSPP from https://mega.nz/folder/pVBmECZJ#YLUEY20mw9d5udHMMibvQQ (backed up by @MojoJojoDojo )
> For the missing exe (v1.11.3-930 to 951) you can use this https://sam.nl.tab.digital/s/AGYbdSMQM783LR8

Thanks bro, I'll try and update the post whether it work or not

**Comment by ItsukaHiro** [comment]
> Based on https://discord.com/channels/293316141479362560/480103153803460614/949986949106647132 Someone said PPSSPP have issue with DLC on Gladiator Begins after [87dcf60](https://github.com/hrydgard/ppsspp/commit/87dcf60ed037d5f2168d5f0033dda002b098e177)
> 
> So you may want to try PPSSPP version before `v1.11.3-942-g87dcf60ed` in case this game also had similar issue too. You can download old version of PPSSPP from https://mega.nz/folder/pVBmECZJ#YLUEY20mw9d5udHMMibvQQ (backed up by @MojoJojoDojo ) For the missing exe (v1.11.3-930 to 951) you can use this https://sam.nl.tab.digital/s/AGYbdSMQM783LR8

Bro, it doesn't work :(((

**Comment by MojoJojoDojo** [comment]
A bit off-topic but there is a Duke Nukem game that got leaked that got the same issue here.
Black screen unless you use PPSSPP 1.11

**Comment by Gamemulatorer** [comment]
> A bit off-topic but there is a Duke Nukem game that got leaked that got the same issue here. Black screen unless you use PPSSPP 1.11

I tried this game and it cause to freeze ppsspp after loading.
![Screenshot_2022-04-16-23-06-57-783_org ppsspp ppsspp](https://user-images.githubusercontent.com/37603562/163680236-1fad6c56-94e4-4d26-939f-0894ff0e8e42.jpg)

According to the description this game requires this.
![IMG_20220416_230806](https://user-images.githubusercontent.com/37603562/163680263-0899e64a-b463-436a-8a56-f543ecbc49f4.jpg)

**Comment by ANR2ME** [comment]
> > A bit off-topic but there is a Duke Nukem game that got leaked that got the same issue here. Black screen unless you use PPSSPP 1.11
> 
> I tried this game and it cause to freeze ppsspp after loading. ![Screenshot_2022-04-16-23-06-57-783_org ppsspp ppsspp](https://user-images.githubusercontent.com/37603562/163680236-1fad6c56-94e4-4d26-939f-0894ff0e8e42.jpg)
> 
> According to the description this game requires this. ![IMG_20220416_230806](https://user-images.githubusercontent.com/37603562/163680263-0899e64a-b463-436a-8a56-f543ecbc49f4.jpg)

Yes, You need to use 64mb memory, you can create a dummy plugin to set the memory to 64mb, like this `plugin.ini`:
```[options]
version = 1
memory = 64

[games]
ULES-01391 = true
NPEZ00324 = true
```

**Comment by hrydgard** [comment]
Hm, I'll do something so that they'll work by default, instead. Thanks for reporting the game IDs.

**Comment by ANR2ME** [comment]
There were 3 games that seems to be based on the same development sources, but i only tried 2 of it.
But @MojoJojoDojo seems to have tried all 3 of them, may be he can provide the game ID for that other game (if it also need 64mb too that is)

**Comment by hrydgard** [comment]
OK, I got the data for Duke and Extraction Point, fixing in #15484 .

However that won't fix this issue about Danball Senki.

**Comment by Gamemulatorer** [comment]
This issue is a duplicate https://github.com/hrydgard/ppsspp/issues/5918#issuecomment-778162110

**Comment by Gamemulatorer** [comment]
What about now?

**Comment by hrydgard** [comment]
Closing because this is part of #5918
